src/test/java/uk/gov/hmcts/reform/sandl/snlevents/service/HearingPartServiceTest.java

```diff
@@ -275,6 +277,35 @@ private Session createSession() {
         return session;
     }
 
+    private StatusConfig createStatusConfigListed() {
+        val statusConfig = new StatusConfig();
```

Comment:
@Import({StatusesTestConfiguration.class})
gives you autowire access to all prepopulated StatusConfig values setup in StatusesMock class